bats/fb-verify-selinux.bats

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+
+@test "ensure no SELinux denials" {
+  if tIsRedHatCompatible; then
+    run ausearch --message AVC
+    echo "$output"
+    [ "${status}" -eq 1 ]
```

Comment:
Why did you choose exit code 1? Accoding to `man ausearch` this is:
> if nothing found, or argument errors or minor file acces/read errors

Reply:
correct, I want it not to find anything, because if it finds something, there were denials.

you could also `-ne 0` ;)